packages/core/src/discovery.js

```diff
@@ -359,15 +362,29 @@ export function createDiscoveryQueue(percy) {
         });
       });
     })
-    .handle('error', ({ name, meta }, error) => {
+    .handle('error', async ({ name, meta }, error) => {
```

Comment:
Any impact due to async 🤔

Reply:
Ignore this, we do await in generatePromise so this should be fine, just run few builds to confirm everything is working fine

Reply:
- Initially, I thought of making it without async, the problem here is:
1. The message is not displayed at the proper location.
2. In some cases, even if the script exists, the request is yet not resolved.

Putting here async will not cause an issue, as we already await similarly at sendComparison, sendSnapshot, createSnapshot etc, in case of network request fails it is caught in `suggestForFix` only, so impacts here.